src/index.js

```diff
@@ -1,42 +1,87 @@
+import React, { useEffect, useState } from "react";
+import ReactDOM from "react-dom";
 import { getNextGameboard, isFieldValueAlive } from "./gameboard";
 
-function renderGameboard(gameboard) {
-  return `<div class="gameboard">${gameboard
-    .map(
-      (row) =>
-        `<div class="gameboard-row">${row
-          .map(
-            (field) =>
-              `<div class="gameboard-field"><div class="gameboard-cell gameboard-cell--${
-                isFieldValueAlive(field) ? "alive" : "dead"
-              }"></div></div>`
-          )
-          .join("")}</div>`
-    )
-    .join("")}</div>`;
+// function renderGameboard(gameboard) {
+//   return `<div class="gameboard">${gameboard
+//     .map(
+//       (row) =>
+//         `<div class="gameboard-row">${row
+//           .map(
+//             (field) =>
+//               `<div class=""><div class="gameboard-cell gameboard-cell--${
+//                 isFieldValueAlive(field) ? "alive" : "dead"
+//               }"></div></div>`
+//           )
+//           .join("")}</div>`
+//     )
+//     .join("")}</div>`;
+// }
+
+function Gameboard({ gameboard }) {
+  return (
+    <div className="gameboard">
+      {gameboard.map((gameboardRow, rowIndex) => {
+        return (
+          <div className="gameboard-row" key={`${rowIndex}`}>
+            {gameboardRow.map((gameboardField, columnIndex) => (
+              <div
+                className="gameboard-field"
+                key={`${rowIndex}-${columnIndex}`}
+              >
+                <Cell field={gameboardField}></Cell>
+              </div>
+            ))}
+          </div>
+        );
+      })}
+    </div>
+  );
 }
 
-function render(gameboard) {
-  document.getElementById("app").innerHTML = `
-  ${renderGameboard(gameboard)}
-`;
+function Cell({ field }) {
+  return (
+    <div
+      className={`gameboard-cell gameboard-cell--${
+        isFieldValueAlive(field) ? "alive" : "dead"
+      }`}
+    ></div>
+  );
 }
 
-function play() {
-  let gameboard = [
-    [0, 0, 0, 0, 0, 0],
-    [0, 0, 1, 0, 0, 0],
-    [0, 0, 1, 0, 0, 0],
-    [0, 0, 1, 0, 0, 0],
-    [0, 0, 0, 0, 0, 0],
-  ];
-
-  render(gameboard);
-  setInterval(() => {
-    gameboard = getNextGameboard(gameboard);
-    render(gameboard);
-  }, 1000);
+const initialGameboard = [
+  [0, 0, 0, 0, 0, 0],
+  [0, 0, 1, 0, 0, 0],
+  [0, 0, 1, 0, 0, 0],
+  [0, 0, 1, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0],
+];
+
+function App() {
+  const [gameboard, setGameboard] = useState(initialGameboard);
+  const [test, setTest] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setGameboard((prevGameboard) => getNextGameboard(prevGameboard));
+      console.log("DEBUG This will run every second!");
+    }, 1000);
+    return () => clearInterval(interval);
+  }, []);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setSeconds((seconds) => seconds + 1);
+    }, 1000);
+    return () => clearInterval(interval);
+  }, []);
+
```

Comment:
using two useEffect is quite nice in regards of seperation of concerns and if your are not worry about performance and a lot of rerenders. If performance becomes an problem you could think about merging the two useEffects as they are using the exact same setInterval, and you can execute your two state updates from within or event create on state object with both values and update this only once